### DISCUSSION TITLE
2024/02/28/Openp2p-Launcher-MC%E5%B8%AE%E5%8A%A9%E6%96%87%E6%A1%A3/

### DISCUSSION THREAD

**Discussion by giscus** [Guailoudou/pages#1] [Announcements]
# 2024/02/28/Openp2p-Launcher-MC%E5%B8%AE%E5%8A%A9%E6%96%87%E6%A1%A3/

github pages

http://localhost:4000/2024/02/28/Openp2p-Launcher-MC%E5%B8%AE%E5%8A%A9%E6%96%87%E6%A1%A3/

**Comment by Guailoudou** [comment]
阿巴阿巴，这个排版怪怪的嘞

**Reply by Guailoudou** [reply]
我的评价是，能用就行

**Reply by Guailoudou** [reply]
优化了半天，舒服多了
